src/main/java/salesforce/ui/pages/CasesFormPage.java

```diff
@@ -21,8 +21,42 @@
     private WebElement caseOriginMenu;
     @FindBy(xpath = "//span[text()='Priority']/../..//a[@class='select']")
     private WebElement priorityMenu;
+    @FindBy(xpath = "//span[text()='Type']/../..//a[@class='select']")
+    private WebElement typeMenu;
+    @FindBy(xpath = "//span[text()='Case Reason']/../..//a[@class='select']")
+    private WebElement caseReasonMenu;
+    @FindBy(xpath = "//span[text()='Product']/../..//a[@class='select']")
+    private WebElement productMenu;
+    @FindBy(xpath = "//span[text()='Potential Liability']/../..//a[@class='select']")
+    private WebElement potentialLiabilityMenu;
+    @FindBy(xpath = "//span[text()='SLA Violation']/../..//a[@class='select']")
+    private WebElement slaViolationMenu;
+    @FindBy(xpath = "//label[text()='Status']/../..//input[@role='combobox']")
+    private WebElement statusMenu;
     @FindBy(xpath = "//span[text()='Contact Name']/../..//input")
     private WebElement contactNameTextBox;
+    @FindBy(xpath = "//span[text()='Account Name']/../..//input")
```

Comment:
Locators with the same structure and just a different value could be set in a generic string and use your get webelement by xpath and value method.